portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -93,6 +102,25 @@ definition {
 		}
 	}
 
+	macro choosePreselect {
```

Comment:
I would call this "selectPreselectedValue"

Reply:
Edited macro name

Reply:
Done!